phac-nml/irida-next
# Feature: Transfer samples

Fixes #84.

## What does this PR do and why?
This PR only covers transfers samples to another existing project, not a new project. 
This feature is restricted to owners.

Note: CSS not included.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/d69c1178-9c6d-46a2-bf66-6c39a38b71a4)
![image](https://github.com/phac-nml/irida-next/assets/325703/5747d4c1-5a2f-44c5-98cb-bb3341b24173)

## How to set up and validate locally
1. Navigate to a project samples page that the logged in user is an owner of.
2. Check the samples you wish to transfer.
3. Hit the "Transfer samples" button.
4. A modal will appear with a project drop-down. Select the project you wish to transfer the samples to.
5. Verify the samples transferred to the target project.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
